_opensearch/geohexgrid-agg.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: default
+title: GeoHex grid aggregations
+parent: Aggregations
+nav_order: 4
+has_children: false
+---
+
+# GeoHex grid aggregations
+
+The Hexagonal hierarchical geospatial indexing system (H3) partitions the Earth's areas into identifiable hexagon-shaped cells. 
+
+The H3 grid system works well for proximity applications, because it overcomes the shortcomings of geohash's non-uniform partitions. Geohash encodes latitude and longitude pairs, leading to significantly smaller partitions near the poles and a degree of longitude near the equator. However, the H3 grid system's distortions are low and limited to five partitions of 122. These five partitions are placed in low-use areas (for example, in the middle of the ocean), leaving the essential areas error-free. Thus, grouping documents based on the H3 grid system provides a better aggregation than the geohash grid.
+
+The GeoHex grid aggregation groups [geopoints]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/) into grid cells for geographical analysis. Each grid cell corresponds to an [H3 cell](https://h3geo.org/docs/core-library/h3Indexing/#h3-cell-indexp) and is identified using the [H3Index representation](https://h3geo.org/docs/core-library/h3Indexing/#h3index-representation).
+
+## Precision
+
+The `precision` parameter controls the level of granularity that determines the grid cell size. The lower the precision, the larger the grid cells. 
+
+The following example illustrates low-precision and high-precision aggregation requests.
+
+To start, create an index and map the `location` field as a `geo_point`:
+
+```json
+PUT national_parks
+{
+  "mappings": {
+    "properties": {
+      "location": {
+        "type": "geo_point"
+      }
+    }
+  }
+}
+```
+
+Index the following documents into the sample index:
+
+```json
+PUT national_parks/_doc/1
+{
+  "name": "Yellowstone National Park",
+  "location": "44.42, -110.59" 
+}
+
+PUT national_parks/_doc/2
+{
+  "name": "Yosemite National Park",
+  "location": "37.87, -119.53" 
+}
+
+PUT national_parks/_doc/3
+{
+  "name": "Death Valley National Park",
+  "location": "36.53, -116.93" 
+}
+```
+
+You can index geopoints in several formats. For a list of all supported formats, see the [geopoint documentation]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point#formats). 
+{: .note}
+
+## Low-precision requests
+
+Run a low-precision request that should bucket all three documents together:
```

Comment:
Should this read "..that buckets all..."

Reply:
This is what's expected to happen after you run this request.

Reply:
I agree that "should" reads strangely here and would go with @vagimeli suggestion, if accurate.